src/main/java/com/zufar/onlinestore/payment/converter/StripeCustomerConverter.java

```diff
@@ -0,0 +1,44 @@
+package com.zufar.onlinestore.payment.converter;
+
+import com.stripe.param.CustomerCreateParams;
+import com.zufar.onlinestore.user.entity.Address;
+import com.zufar.onlinestore.user.entity.UserEntity;
+import org.apache.commons.lang3.StringUtils;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingConstants;
+import org.springframework.stereotype.Component;
+import java.util.Map;
+
+@Component
+@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)
```

Comment:
maybe better use "spring" in the componentModel instead of MappingConstants.ComponentModel.SPRING ?